hostscripts/zuul/layout.yaml

```diff
@@ -38,12 +38,13 @@ projects:
       - openstack-rpm-packaging-sles12-Ocata
       - openstack-rpm-packaging-sles12-Pike
       - openstack-rpm-packaging-sles12-Queens
+      - openstack-rpm-packaging-sles12-Rocky
     post-rpm-packaging:
       - openstack-rpm-packaging-update-Master
       - openstack-rpm-packaging-update-Newton
       - openstack-rpm-packaging-update-Ocata
       - openstack-rpm-packaging-update-Pike
-      - openstack-rpm-packaging-update-Queens
+      - openstack-rpm-packaging-update-Rocky
```

Comment:
iiuc this will prevent update jobs being triggered after something is merged in Queens